packages/dapp/src/components/organisms/CreativePanel.tsx

```diff
@@ -112,11 +112,31 @@ export const CreativePanel = ({
   const onNFTImageLoaded = useCallback(
     (event: SyntheticEvent<HTMLImageElement, Event>): void => {
       if (spliceDataUrl || dominantColors?.length > 0) return;
-      console.log('extract');
-      extractColors(event.currentTarget, {}).then((colors) => {
+
+      const xOptions = {
+        proxy: process.env.REACT_APP_CORS_PROXY
+      };
+      const onExtracted = (colors: RGB[]) => {
         setDominantColors(colors);
         if (onDominantColors) onDominantColors(colors);
-      });
+      };
+
+      extractColors(event.currentTarget, xOptions)
+        .then(onExtracted)
+        .catch((e: any) => {
+          console.log(
+            'extracting failed, trying again with cors proxy. Reason:',
+            e.message
+          );
+          extractColors((event as any).path[0].src, xOptions)
```

Comment:
this actually can fail (event.path is potentially undefined, for whatever reasons), e.g.: https://bafybeievbr6omeydrszk3t2hivpwovvqrfb4zagy7aizq433eamttvyi4i.on.fleek.co/#/nft/0xF5aa8981E44a0F218B260C99F9C89Ff7C833D36e/5 by 0xAEe972750B169d537c0087861bffFCfC78E99cd7